src/Microsoft.Identity.Web.UI/Microsoft.Identity.Web.UI.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1; net5.0</TargetFrameworks>
+    <TargetFrameworks>net6.0; net7.0; net8.0</TargetFrameworks>
```

Comment:
can you explain a bit more?

Reply:
![image](https://github.com/AzureAD/microsoft-identity-web/assets/69649063/6c492c8f-8afe-4c8d-9449-73a5d012e69a)
^ With only .net5

![image](https://github.com/AzureAD/microsoft-identity-web/assets/69649063/cc3b2ff1-e120-4839-a4f7-e44d18823b98)
^ With .net5, .net6, .net7. ,net8

And with .net6, .net7, .net8 there are no errors :/

Reply:
+ this may be appropriate since we're moving to IdWeb 3x

Reply:
we need to check if this is still an issue:
https://github.com/AzureAD/microsoft-identity-web/issues/2183 
before making this change

Reply:
and this needs to go through a release build (as the issue was only seen on signed code)